include/unifex/sender_concepts.hpp

```diff
@@ -179,14 +179,9 @@ UNIFEX_CONCEPT //
     detail::_has_sender_types<sender_traits<remove_cvref_t<S>>>;
 
 template <typename S>
-UNIFEX_CONCEPT   //
-  typed_sender = //
```

Comment:
Assuming it's even possible, it might be a good idea to mark this deprecated instead of deleting it.

Reply:
Concepts don't get attributes until C++23. But you can make it a Boolean variable template.